frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1308,6 +1316,15 @@ private void prepareProjectEdit(RenderRequest request) {
                 project = client.getProjectByIdForEdit(id, user);
                 usingProjects = client.searchLinkingProjects(id, user);
                 allUsingProjectCount = client.getCountByProjectId(id);
+            } catch (SW360Exception sw360Exp) {
+                if (sw360Exp.getErrorCode() == 404) {
```

Comment:
move this logic to some common method and reuse it.

Reply:
Changes Made